scripts/qiita

```diff
@@ -59,9 +62,24 @@ def pet():
     pass
 
 
-@pet.group()
-def webserver():
-    pass
+@pet.group(invoke_without_command=True)
```

Comment:
The reason why I didn't make it part of the start command, is because
you would not be able to just build the documentation if you wanted to,
you would be forced to start the server every time you wanted to build
the docs. I'm open to suggestions on this end though.

On (Jun-08-15| 9:54), josenavas wrote:
> > @@ -59,9 +62,24 @@ def pet():
> >      pass
> >  
> >  
> > -@pet.group()
> > -def webserver():
> > -    pass
> > +@pet.group(invoke_without_command=True)
> 
> Should this instead be part of the 'start' command? IMO it's a bit weird that we are actually calling a group rather than the command.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1191/files#r31935072